octobot_pro/resources/reports/header.html

```diff
@@ -0,0 +1,30 @@
+<head>
+    <title>{{ title }} OctoBot pro report</title>
+
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <link rel="shortcut icon" type="image/png" href="https://octobot.online/assets/images/octobot_30px.png">
+
+    <!-- Prevent search engines indexing -->
+    <meta name="robots" content="noindex">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+
+    <!-- Fontawesome CSS -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v6.2.1/css/all.css" integrity="sha384-twcuYPV86B3vvpwNhWJuaLdUSLF9+ttgM2A6M870UYXrOsxKfER2MKox5cirApyA" crossorigin="anonymous">
+
+    <!-- mdb CSS -->
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha384-VB0/w/SA4yj77RioXx1gUta4sczGrqbM4McYM8Lbixo0bQjK270LMDrUcp12DETJ" rel="stylesheet" crossorigin="anonymous">
```

Comment:
Do we use MDB components in OctoBot-Pro?

Reply:
we don't, i'm removing it